portfolio/src/main/webapp/index.html

```diff
@@ -182,6 +182,8 @@ <h4 class="card-title">Portfolio</h4>
         </div>
         <!-- Comments Section -->
         <div class="tab-pane container" id="comments">
+          <div id="comment-posted-status">
```

Comment:
nit: <div id="comment-posted-status"/>?

Reply:
I'm not too sure if that is allowed. I've read (mostly on stackoverflow) that self-closing div tags aren't technically valid HTML5 but they would be valid in XML.

https://stackoverflow.com/questions/3852469/is-it-valid-html5-to-use-a-single-tag-for-a-div/3852501
https://stackoverflow.com/questions/3558119/are-non-void-self-closing-tags-valid-in-html5

Reply:
I see. Thanks for checking.